modules/apps/commerce/commerce-account-service/src/main/java/com/liferay/commerce/account/internal/util/AccountEntryUtil.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.account.internal.util;
+
+import com.liferay.commerce.context.CommerceContext;
+import com.liferay.commerce.product.model.CommerceChannel;
+import com.liferay.commerce.product.service.CommerceChannelLocalServiceUtil;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.model.Group;
+
+/**
+ * @author Crescenzo Rega
+ */
+public class AccountEntryUtil {
+
+	public static long getCommerceChannelId(
+			CommerceContext commerceContext, Group group)
+		throws PortalException {
+
+		long commerceChannelId = 0;
```

Comment:
don't need to declare this variable, just return the value when you get it

Reply:
I made changes